test/any.test.js

```diff
@@ -227,5 +237,15 @@ test('should throw a TypeError with the path to the key of the invalid value /2'
 
   const stringify = build(schema)
 
-  t.throws(() => stringify({ data: { kind: 'Baz', value: 1 } }), new TypeError('The value of \'#/properties/data\' does not match schema definition.'))
+  try {
+    stringify({ data: { kind: 'Baz', value: 1 } })
+    t.fail('should throw')
+  } catch (err) {
+    t.equal(err.message, 'The value of \'#/properties/data\' does not match schema definition.')
+    t.same(err.validationErrors, [
```

Comment:
dito
```suggestion
    t.same(err.cause, [
```

Reply:
i think the rationale here was to call it as fastify with ajv implementation:

https://github.com/fastify/fastify/blob/9b3ca58097954c4df20b253906ed6697461e2486/lib/handleRequest.js#L109